src/logic/basic.lean

```diff
@@ -946,6 +946,12 @@ theorem exists_eq {a' : α} : ∃ a, a = a' := ⟨_, rfl⟩
 
 @[simp] theorem exists_eq' {a' : α} : ∃ a, a' = a := ⟨_, rfl⟩
 
+theorem exists_unique_eq {a' : α} : ∃! a, a = a' :=
```

Comment:
Does the linter complain if this is `simp`? That's the reason that `exists_eq` is not `simp`.